src/exceptions.py

```diff
@@ -0,0 +1,17 @@
+def fun_TypeError(num1, num2, flag):
```

Comment:
please use camel case. Your code looks good, but this is the common issue, please contact me if you have any 
questions